wiki/en/en-Client-Troubleshooting.md

```diff
@@ -18,11 +18,10 @@ Start with the simple stuff: make sure your instrument/microphone and headphones
 
 You can test whether you are hearing your signal correctly by doing the following:
 
-1.  Connect your audio source (mic, instrument...) to your computer.
-1.  Use an audio recording application (such as [Audacity](https://www.audacityteam.org/)) on your computer to record that input and check it's OK.
-1.  Make sure that if you mute your audio input inside the recording application, you can't hear yourself.
-1.  Close the recording application and run Jamulus. You should still not hear yourself.
-1.  Connect to a server and play something. Now you should hear yourself with the delay from the server.
+1. Run Jamulus and connect to a server with a long ping time (greater than 200ms)
+2. Clap your hands once (or play one short note on an instrument). You should hear the sound from the Jamulus server come back, but significantly delayed.
```

Comment:
Oh dear.  So it doesn't understand markdown.  That's not helpful.  At least it's stopped moaning about spaces..!  (I assume it wanted each "1" indented under the previous - makes some sense if it didn't understand they'd auto-number.)